### ISSUE TITLE
Allow lists in @input-border-width

### ISSUE THREAD

**Issue by alawatthe** [jquense/react-widgets#346]
Currently it is only possible to use one single value for the less variable ```@input-border-width```.
I would like that it is possible to use lists of one to four values, such that I can style the ```border-bottom-width``` differently from the other border-widths.

Since the variable is used not very frequently (https://github.com/jquense/react-widgets/search?utf8=%E2%9C%93&q=%22%40input-border-width%22&type=Code), the fix would not be very complicated, and would not change the behavior of existing code.

Basically https://github.com/jquense/react-widgets/blob/master/src/less/core.less#L81 and 
https://github.com/jquense/react-widgets/blob/master/src/less/core.less#L263 need to be changed from 
```less
border: @input-border @input-border-width solid;
```
to
```less
border: @input-border solid;
border-width: @input-border-width;
```

Only the lines https://github.com/jquense/react-widgets/blob/master/src/less/popup.less#L32-L33 need a slightly more complicated fix.

Shall I send a pull request?

**Comment by jquense** [comment]
A PR is always appreciated , we just need to be careful (as you note) to not change the existing default behavior, don't want to break anyone.

**Comment by alawatthe** [comment]
I had a look at this and unfortunately the https://github.com/jquense/react-widgets/blob/master/src/less/popup.less#L32-L33 problem is more difficult then I anticipated. I would need to extract the ```left``` and ```right``` value from the list, which can be done with something like this:

```less
.left (@list) when (length(@list) = 1) {
  left: -10px - extract(@list, 1);
}
.left (@list) when (length(@list) = 2) {
  left: -10px - extract(@list, 2);
}
.left (@list) when (length(@list) = 3) {
  left: -10px - extract(@list, 2);
}
.left (@list) when (length(@list) = 4) {
  left: -10px - extract(@list, 4);
}

.right (@list) when (length(@list) = 1) {
  right: -10px - extract(@list, 1);
}
.right (@list) when (length(@list) = 2) {
  right: -10px - extract(@list, 2);
}
.right (@list) when (length(@list) = 3) {
  right: -10px - extract(@list, 2);
}
.right (@list) when (length(@list) = 4) {
  right: -10px - extract(@list, 2);
}

.rw-popup-container {
  // ...
  left(@input-border-width);
  right(@input-border-width);
```
Which is way to much overhead for this task imho.

I'll think a little more on this problem. Perhaps I find an easier solution.

**Comment by alawatthe** [comment]
Sadly, I don't think that there is an easy solution :-/
